src/Controller.h

```diff
@@ -73,9 +73,7 @@ class Controller
   void indicateMode(byte mode);
   void indicateActivity();
   void setLearnMode(byte reqMode);
-  bool isSetProdEventTableFlag() { return setProdEventTable; }
-  void clearProdEventTableFlag();
```

Comment:
Should you also get rid of the flag setProdEventTable ?

Reply:
Yes.  I'll update